tests/config/test_defaults.py

```diff
@@ -29,7 +29,6 @@ def test_create_defaults() -> None:
     """Test dumping the default values."""
     output_dir = os.path.dirname(os.path.abspath(__file__))
     assert create_defaults(output_dir, global_config.macaron_path) is True
-    assert create_defaults("/", "/") is False
```

Comment:
This particular test has been helpful in finding bugs related to permission issues in the past. Instead of removing it, please replace it by a testcase that raises `OSError` [here](https://github.com/oracle/macaron/blob/0f426982837708539fb64b76001b823b79596b0e/src/macaron/config/defaults.py#L166).

Reply:
Do you have any suggestions for improving this test?
One solution I can think of now is to detect if the current user running tests is `root`, and disable the test if it is the case.

Reply:
I fixed the test in 268839891f7c4719bbe1339038a4652d90ee38b2.
I set it to expect to fail in case the user is `root`.

Reply:
The adjustment to the test looks good.